hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastProxyFactory.java

```diff
@@ -436,7 +436,7 @@ public static ProxySource of(Class<?> klass, ClassLoader targetClassLoader) {
         public static final AllAsPublicConstructorStrategy INSTANCE = new AllAsPublicConstructorStrategy();
 
         @Override
-        public MethodRegistry inject(MethodRegistry methodRegistry) {
+        public MethodRegistry inject(TypeDescription instrumentedType, MethodRegistry methodRegistry) {
```

Comment:
Is this leftover?

Reply:
It's not a leftover, the ByteBuddy API has changed in the new versions.